docs/application/dotnet/guides/app-management/widget-control.md

```diff
@@ -0,0 +1,151 @@
+
+# Widget Control
+
+The widget control provides information about installing and running widget applications.
+
+It also provides functions for the following:
+
+*   Sending update requests to the widget applications
+*   Retrieving details of running instance for the same package widget applications
+
+
+The main features of the `Tizen.Applications.WidgetControl` class include:
+
+-   [Getting information on widget applications](#getting_information)
+
+    For installed (but not necessarily running) widget applications, you can retrieve widget information with the [Tizen.Applications.WidgetControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.WidgetControl.html) class.
```

Comment:
From line 16 to line 26.  For the 3 list items, we have sentences end with "with the Tizen.Applications.WidgetControl class.".

However as we have links to Tizen.Applications.WidgetControl in the following contents, and as the list itself is about the "Tizen.Applications.WidgetControl class" as written in line 12,
how about removing these 3 "with the Tizen.Applications.WidgetControl class."s?

Reply:
@mijin85cho, sure its a good suggestion to remove "with the Tizen.Applications.WidgetControl class."
@gotoextreme, please remove "with the Tizen.Applications.WidgetControl class" as suggested

Reply:
Done